components/threading/global_threads/thread_list/virtualized_thread_list.tsx

```diff
@@ -46,7 +50,15 @@ function VirtualizedThreadList({
         }
     }, [selectedThreadId, ids]);
 
-    const data = useMemo(() => ({ids, selectedThreadId}), [ids, selectedThreadId]);
+    const data = useMemo(
+        () => (
+            {
+                ids: hasLoaded && ids.length === total ? [...ids, Constants.THREADS_NO_RESULTS_ITEM_ID] : (isLoading && ids.length !== total && [...ids, Constants.THREADS_LOADING_INDICATOR_ITEM_ID]) || ids,
```

Comment:
I saw an issue where the first attempt to load more wouldn't show the loading indicator, but subsequent loads would. I'm guessing its something related to the `isLoading` check here.

https://user-images.githubusercontent.com/13738432/146991231-26e44bac-7e9b-475f-9c08-c829990860a7.mp4